configure.ac

```diff
@@ -2934,6 +2936,16 @@ AC_CHECK_TYPE(cpu_set_t,
 #endif
 ])
 
+AC_CHECK_TYPE(cpuset_t,
+   AC_DEFINE(HAVE_CPUSET_T,1,[cpuset_t is defined by the system headers]),,[
```

Comment:
AFAICT, HAVE_CPUSET_T is currently unused in this PR (but it probably should be used!).

Reply:
It does